File: EscapeFromTrinityEngineStats/Models/Relationships/ShopAffordablePassive.cs
using EscapeFromTrinityEngineStats.Models.Instances;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EscapeFromTrinityEngineStats.Models.Relationships
{
    public class ShopAffordablePassive
    {
        public int ShopRecordId { get; set; }
        public ShopRecord ShopRecord { get; set; }

        public int PassiveInstanceId { get; set; }
        public PassiveInstance PassiveInstance { get; set; }
    }
}

File: EFTEDashboard/src/app/interfaces/relationships/shop-affordable-passive.ts
import { PassiveInstance } from '../Instances/passive-instance';
import { ShopRecord } from '../shop-record';

export interface ShopAffordablePassive {
	shopRecordId?: number;
	shopRecord?: ShopRecord;
	passiveInstanceId?: number;
	passiveInstance?: PassiveInstance;
}
